server/routes/healthy.js

```diff
@@ -1,11 +1,22 @@
 const dbService = require('../services/database-service')
+const dbVersion = require('../dbversion')
 
 module.exports = {
   method: 'GET',
   path: '/healthy',
   options: {
     handler: async (request, h) => {
-      if (await dbService.isConnected()) {
+      let allOk = false
+      try {
+        if (await dbService.isConnected()) {
+          if (await dbVersion.versionCorrect()) {
+            allOk = true
+          }
+        }
+      } catch (error) {
+        allOk = false
+      }
+      if (allOk) {
         return h.response('ok').code(200)
       } else {
         return h.response('not ok').code(500)
```

Comment:
It would be nice to get a list of which checks failed to assist debugging when a readiness check fails.

Also, as @markysoft pointed out to me when I did the `/healthy` route for the Claim service, it's good practise to use constants, rather than `var` or `let`, as much as possible to avoid accidental variable overwrites in later code changes.

Check the [claim service `/healthy` route](https://github.com/DEFRA/ffc-demo-claim-service/blob/master/server/routes/healthy.js) for an example of similar checks taking the above into account.